editor/src/components/filebrowser/fileitem.tsx

```diff
@@ -572,20 +563,17 @@ class FileBrowserItemInner extends React.PureComponent<
     const extraIndentationForExport = 0
     const indentation = this.state.pathParts.length + extraIndentationForExport - 1
 
-    const regularBackground = this.state.isHovered
-      ? colorTheme.neutralBackground.value
-      : 'transparent'
-    const dropTargetBackground = colorTheme.listDropTargetBackground
-
-    const dropTargetOutline = isCurrentDropTargetForAnyFiles
-      ? {
-          boxShadow: `inset 0px 0px 0px 1px ${colorTheme.listDropTargetOutline.value}`,
-        }
-      : {}
-
-    const resultingBackground = isCurrentDropTargetForAnyFiles
-      ? dropTargetBackground
-      : regularBackground
+    const getBackground = () => {
+      let resultingBackground = 'transparent'
+      if (this.props.isSelected) {
+        resultingBackground = colorTheme.subtleBackground.value
+      } else if (this.state.isHovered) {
+        resultingBackground = colorTheme.secondaryBackground.value
+      } else if (isCurrentDropTargetForAnyFiles) {
+        resultingBackground = colorTheme.brandNeonYellow.value
+      }
+      return resultingBackground
```

Comment:
There's no need for the `let` and the variable assignment here - each statement could just `return` the value instead (with the final `else` returning the default `'transparent'` value)

Reply:
Fixed